src/tigerbeetle/main.zig

```diff
@@ -446,6 +447,110 @@ const Command = struct {
+                    // It would be difficult to gather the addresses of non-heap memory that is also
+                    // faulted in, such as the stack, globals, etc. SetProcessWorkingSetSize can be
+                    // used instead to lock all existing pages into memory to avoid swapping.
+                    const set_process_working_set_size = @extern(
+                        *const fn (
+                            hProcess: os.windows.HANDLE,
+                            dwMinimumWorkingSetSize: os.windows.SIZE_T,
+                            dwMaximumWorkingSetSize: os.windows.SIZE_T,
+                        ) callconv(os.windows.WINAPI) os.windows.BOOL,
+                        .{
+                            .library_name = "kernel32",
+                            .name = "SetProcessWorkingSetSize",
+                        },
+                    );
+                    const get_process_working_set_size = @extern(
+                        *const fn (
+                            hProcess: os.windows.HANDLE,
+                            lpMinimumWorkingSetSize: *os.windows.SIZE_T,
+                            lpMaximumWorkingSetSize: *os.windows.SIZE_T,
+                        ) callconv(os.windows.WINAPI) os.windows.BOOL,
+                        .{
+                            .library_name = "kernel32",
+                            .name = "GetProcessWorkingSetSize",
+                        },
+                    );
+
+                    const process_handle = os.windows.kernel32.GetCurrentProcess();
+                    var working_set_min: os.windows.SIZE_T = 0;
+                    var working_set_max: os.windows.SIZE_T = 0;
+
+                    if (get_process_working_set_size(
+                        process_handle,
+                        &working_set_min,
+                        &working_set_max,
+                    ) == os.windows.FALSE) {
+                        working_set_min = counting_allocator.size; // Count bytes allocated so far.
+                        working_set_min += 64 * 1024 * 1024; // 64mb buffer room for stack/globals.
+                        working_set_max = working_set_min * 2; // Buffer room for new page faults.
+                    }
+
+                    if (set_process_working_set_size(
+                        process_handle,
+                        working_set_min,
+                        working_set_max,
+                    ) == os.windows.FALSE) {
+                        // From std.os.windows.unexpectedError():
+                        const fmt_flags = os.windows.FORMAT_MESSAGE_FROM_SYSTEM |
+                            os.windows.FORMAT_MESSAGE_IGNORE_INSERTS;
+
+                        // 614 is the length of the longest windows error description
+                        var buf_wstr: [614:0]os.windows.WCHAR = undefined;
+                        const len = os.windows.kernel32.FormatMessageW(
```

Comment:
```suggestion
                        const size = os.windows.kernel32.FormatMessageW(
```

Either `size` or `count` or `length`.